pkgs/by-name/bi/bitbox-bridge/package.nix

```diff
@@ -34,6 +34,13 @@ rustPlatform.buildRustPackage (finalAttrs: {
     libudev-zero
   ];
 
+  postInstall = lib.optionals stdenv.hostPlatform.isLinux ''
```

Comment:
```suggestion
  postInstall = lib.optionalString stdenv.hostPlatform.isLinux ''
```

Reply:
@HeitorAugustoLN changes made, thanks for pointing it out! ;)